tests/integration-async/records/conftest.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+import logging
+import time
+from copy import deepcopy
+from datetime import datetime, timedelta
+from functools import partial
+
+import pytest
+from click.testing import CliRunner
+from flask.cli import ScriptInfo
+from helpers.providers.faker import faker
+from inspire_utils.record import get_value
+from invenio_app.factory import create_api as invenio_create_app
+from invenio_db import db
+from invenio_search import current_search_client as es
+
+from inspirehep.records.api import LiteratureRecord
+from inspirehep.records.fixtures import (
+    init_default_storage_path,
+    init_records_files_storage_path,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="session")
+def app():
+    app = invenio_create_app()
+    app_config = {}
+    app_config["CELERY_BROKER_URL"] = "pyamqp://guest:guest@localhost:5672"
+    app_config["CELERY_CACHE_BACKEND"] = "memory"
+    app_config["CELERY_RESULT_BACKEND"] = "cache"
+    app_config["CELERY_TASK_ALWAYS_EAGER"] = False
+    app_config["CELERY_TASK_EAGER_PROPAGATES"] = False
+    app_config["DEBUG"] = True
+    app_config["JSONSCHEMAS_HOST"] = "localhost:5000"
+    app_config["SEARCH_ELASTIC_HOSTS"] = "localhost:9200"
+    app_config[
+        "SQLALCHEMY_DATABASE_URI"
+    ] = "postgresql+psycopg2://inspirehep:inspirehep@localhost:5432/inspirehep"
+    app_config["SQLALCHEMY_TRACK_MODIFICATIONS"] = True
+    app_config["TESTING"] = True
+    app.config.update(app_config)
+    return app
```

Comment:
do you need to configure all these? most of them are configured like the fixture in integration/conftest

Reply:
Here we do not have this fixture, and still this configuration has to be configured somewhere. :)